Source/SwiftLintCore/Models/Issue.swift

```diff
@@ -101,6 +116,20 @@ public enum Issue: LocalizedError, Equatable {
             return "Configuration for '\(id)' rule contains the invalid key(s) \(keys.formatted)."
         case let .invalidRuleIDs(ruleIDs):
             return "The key(s) \(ruleIDs.formatted) used as rule identifier(s) is/are invalid."
+        case let .ruleNotPresentInOnlyRules(id):
+            return "Found a configuration for '\(id)' rule, but it is not present on " +
+                   "'\(Configuration.Key.onlyRules.rawValue)'."
+        case let .ruleDisabledInDisabledRules(id):
+            return "Found a configuration for '\(id)' rule, but it is disabled on " +
+                   "'\(Configuration.Key.disabledRules.rawValue)'."
+        case let .ruleDisabledInParentConfiguration(id):
+            return "Found a configuration for '\(id)' rule, but it is disabled in a parent configuration."
+        case let .ruleNotEnabledInOptInRules(id):
+            return "Found a configuration for '\(id)' rule, but it is not enabled on " +
+                   "'\(Configuration.Key.optInRules.rawValue)'."
+        case let .ruleNotEnabledInParentOnlyRules(id):
+            return "Found a configuration for '\(id)' rule, but it is not present in the parent " +
```

Comment:
Is that more correct?
```suggestion
            return "Found a configuration for '\(id)' rule, but it is not present in the parent's " +
```